packages/maps/src/components/basic/GeoCode.js

```diff
@@ -0,0 +1,46 @@
+import { Component } from 'react';
+
+class GeoCode extends Component {
+	getUserLocation() {
+		navigator.geolocation.getCurrentPosition((location) => {
+			const coordinatesObj = {
+				lat: location.coords.latitude,
+				lng: location.coords.longitude,
+			};
+
+			this.geocoder.geocode({ location: coordinatesObj }, (results, status) => {
+				if (status === 'OK') {
+					if (Array.isArray(results) && results.length) {
+						const userLocation = results[0].formatted_address;
+						this.setState({
+							// eslint-disable-next-line react/no-unused-state
+							userLocation,
```

Comment:
@ShahAnuj2610 Why did you define state if not using it?

Reply:
`setState` gets called in the context of `this` of parent component like `GeoDistanceSlider` and `GeoDistanceDropdown`